src/frontend/views/error.njk

```diff
@@ -1,8 +1,5 @@
 {% extends "common/layout/base.njk" %}
 {% set mainClasses = "govuk-main-wrapper--l" %}
-{% block pageTitle %}
-  {{ headTitle }}
-{% endblock pageTitle %}
```

Comment:
I noticed the error.njk inherits from base.njk, so it was generating the page title twice, which is why I removed it here.  And as that would mean that base.njk is the only place that generates the title now, I thought it made more sense to format the page title there instead of in that function (which I meant to delete -- will do that now).